citations-tool/tool/src/webapp/js/edit_nested_citations.js

```diff
@@ -260,6 +260,7 @@
             if ( !CKEDITOR.instances[sectionInlineEditor] ) {
                 CKEDITOR.inline( sectionInlineEditor, {
                     startupFocus: true,
+                    forcePasteAsPlainText : true,
```

Comment:
Yes it does but you couldn't paste bold/italics before anyway with the PasteAsPlainText button. And we don't want them to use CTRL C+V without stripping it or the Paste button as it pastes in lots of styles and things the ckeditor does not like. ( I'm still working on getting lists working.)